docs/source/docs/explanation/formats/arrow.md

```diff
@@ -0,0 +1,194 @@
+# Arrow
+
+## Format specification
+
+[Apache Arrow](https://arrow.apache.org/docs/format/Columnar.html) is a in-memroy columnar table format specification with multiple langauge support.
+This format supports to export into a single arrow file or multiple shardings.
+
+### Table schema
+
+|   id   | subset | media.type | media.path | media.bytes | media.attributes | annotations | attributes |
+|:------:|:------:|:----------:|:----------:|:-----------:|:----------------:|:-----------:|:----------:|
+| string | string |   uint32   |   string   |    binary   |      binary      |    binary   |   binary   |
+
+### id (`string`)
+The ID of each entity. A tuple of (`id`, `subset`) is a unique key of each entity.
+
+### subset (`string`)
+The subset the entity belongs to. A tuple of (`id`, `subset`) is a unique key of each entity.
+
+### media.type (`uint32`)
+The type of media the entity has.
+
+**Supported media types:**
+
+- 0: `None`
+- 2: `Image`
+- 6: `PointCloud`
+
+### media.path (`string`)
+The path of the media. It could be a real path or a relative path, or `/NOT/A/REAL/PATH` if path is invalid.
+
+### media.bytes (`binary`)
+The binary data of the media. It could be `None` if one chooses not to save media when export.
+
+### media.attributes (`binary`)
+The attribute of the entity. The contents of it depends on `media.type`.
+The byte order is little-endian.
+
+**Image**
+
+![image attributes](../../../../images/arrow/image_attributes.png)
```

Comment:
Why don't we move this image directory to `docs/source/docs/explanation/formats`?

Reply:
I assumed that images are in top level as other images in ‘image’ directory. Do you want me to move them?

Reply:
I think that's cleaner.

Reply:
Moved in https://github.com/openvinotoolkit/datumaro/pull/948/commits/187f2ad1c1f3b46346ba1939d8590911a1bd08d7